.github/workflows/build.yml

```diff
@@ -95,8 +97,3 @@ jobs:
           elif [[ $GITHUB_REF == refs/heads/master ]]; then
             ./goreleaser --snapshot
           fi
-      -
-        name: Clear
-        if: always() && startsWith(github.ref, 'refs/tags/v')
-        run: |
-          rm -f ${HOME}/.docker/config.json
```

Comment:
This is [handled by docker login action](https://github.com/docker/login-action#inputs).